src/i18n/langs/de.json

```diff
@@ -81,6 +81,10 @@
         "lnInvoice": {
           "required": "Bitte gib eine Lightning invoice ein",
           "patternMismatch": "Falsches invoice format"
+        },
+        "chainAddress": {
+          "required": "Bitte gib eine Bitocin Adresse ein",
+          "patternMismatch": "Falsches Adress format"
         }
```

Comment:
`chainFee` missing

Reply:
Missing keys are automatically added if translators add a translation in https://hosted.weblate.org/projects/raspiblitz-web/translations/

So just fill out the english one and weblate takes care of the rest :)

Reply:
Just to remind myself, while on it I will add `en` and `de` but thanks for the heads up!